bower.json

```diff
@@ -25,7 +25,8 @@
         "modernizr": "2.8.3",
         "ng-file-upload": "7.0.17",
         "ngInfiniteScroll": "1.2.0",
-        "swagger-ui": "2.1.2"
+        "swagger-ui": "2.1.2",
+        "amcharts":"3.4.6"
```

Comment:
if you are adding a dependency to package.json, do not include files assosiated with this dependecy. It should be downloaded by npm and should not be in the repository